### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jvoorhis**
s/witch/which/g

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,5 +29,5 @@ This section describe the terms used as along all this document.
   guest root directory. It can be built from any Linux Distribution but
   must provide at least the following components:
 	- Kata agent
-	- A `init` system (for example `systemd`) witch allow to start
+	- A `init` system (for example `systemd`) which allow to start
```
Comment (WeiZhang555) [inline README.md]: `which allows`

**Comment by WeiZhang555** [comment]
Thanks for your PR!  There's only a small nit. :smile:

**Comment by jcvenegas** [comment]
Hi @jvoorhis thank you!. Can you also add  `Signed-off-by` to the commit message?

**Comment by jodh-intel** [comment]
Hi @jvoorhis - I hope you don't mind, but I've resolved the spelling issue on #26, so I'll close this PR...

**Pull request closed without merging**